pkgs/development/compilers/julia/generic-bin.nix

```diff
@@ -85,11 +85,15 @@ stdenv.mkDerivation {
   installPhase = ''
     runHook preInstall
     cp -r . $out
+  '' + lib.optionalString stdenv.hostPlatform.isLinux ''
+    autoPatchelf "$out/bin" "$out/lib" "$out/libexec"
```

Comment:
Maybe add a comment explaining that we're specifically avoiding `$out/share`, where the precompiled files live?